docs/guides/2-cli.md

```diff
@@ -1,24 +1,8 @@
 # Spectral CLI
 
-[Once installed](../getting-started/2-installation.md), Spectral can be run via the command-line:
```

Comment:
I'm not sure whether the expectation here is that the user visits the first page of the guide, but regardless of that
it feels natural to have an example usage here when you visit the CLI page.